ts/de_DE.ts

```diff
@@ -317,7 +317,7 @@
     <message>
         <location filename="gui.py" line="1268"/>
         <source>In what units you want to scale?</source>
-        <translation>In welcher Einheit möchtest du skalieren?</translation>
+        <translation>In welcher Einheit möchten Sie skalieren?</translation>
```

Comment:
Are you sure that the formal adressing is really necessary here? If so then I would definitely suggest doing it at every occasion, there are a lot of questions left with the informal du this way, mixing is the worst of both worlds :)